SCClassLibrary/Common/Core/Object.sc

```diff
@@ -340,8 +362,11 @@ Object  {
 	subclassResponsibility { arg method;
 		SubclassResponsibilityError(this, method, this.class).throw;
 	}
-	doesNotUnderstand { arg selector ... args;
-		DoesNotUnderstandError(this, selector, args).throw;
+	doesNotUnderstand { |selector ...args|
```

Comment:
if we want to make it easy for people to support the interface, Object should delegate: `this.doesNotUnderstandWithKeys(selector, args)` 

Then if someone changes their interface to implement only `doesNotUnderstandWithKeys`, it will still work for *their* subclasses, if they have not yet updated the implementation.